.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,46 +3,21 @@ Thanks for opening a Pull Request (PR), here's a few guidelines as to what we ne
 <!-- REQUIRED  https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue -->
 Fixes: `<issue-number>`
 
-## How did you solve the problem?
+## Are there more details and context that will help us understand and review this pull request?
 
 <!-- REQUIRED
 
-Short description of the changes you made and how they address the linked issue.
+Further description of the changes you made if the pull request title is insufficient, and how they address the linked issue.
```

Comment:
Maybe we should have a separate paragraph about making good titles

Reply:
as one of the checklist items?

Reply:
I don't have a strong opinion on this. @KevinMind any thoughts ?

Reply:
I think it makes sense especially given how we are implicitly expecting the title to be the longest living and most prominent part of the PR in terms of future references.

I've experimented with a simpler syntax in a few PRs, wdyt?

1. Description
2. Context
3. Testing

All optional (begrudgingly) wdyt? I'd like to keep the titles short, even shorter than I originally conceived them.

Ex:
- https://github.com/mozilla/addons-server/pull/21787
- https://github.com/mozilla/addons-server/pull/21784
- https://github.com/mozilla/addons-server/pull/21773
- https://github.com/mozilla/addons-server/pull/21734

@diox could you add a suggestion in the original PR for a section on titles. I think that would be good to explain in the PR template that when you merge the title of the PR will be the commit and how it should be formatted. All great context for a contributor. 

IF we agree on the simplified section names then I would add a commit on the original PR and then @eviljeff could you add your changes as suggestions? Then we will have a finallized version on the original PR and can squash and merge it.